### DISCUSSION TITLE
PocketBook 633 Color

### DISCUSSION THREAD

**Discussion by berdik88** [koreader/koreader#8818] [Q&A] [answered]
Hi!

I installed koreader on my PocketBook 633 Color. 
At the first launch, the [message](https://disk.yandex.ru/i/MQ4IFagj7jLiOA) came out "you have a color screen, everything will be in color, if in reality gray - disable it in the settings." 
But in reality, everything is shown in gray tones. There is no color. I checked the settings, the color checkmark is. 
What am I doing wrong? (((
How do I enable color display? 

The firmware is the latest (20210802_U633_6.4.467). I tried to lower it to the minimum that I found on the website (20201021_U633_6.1.962) - no results, only work has become slower. 
I reset the settings, formatted the memory - there are no results. 

PS: Also, when in night mode, it always makes pictures in the negative, the setting does not affect this.

**Comment by Frenzie** [comment]
Pinging @rjd22

**Comment by rjd22** [comment, marked as answer]
@berdik88 awesome to hear that you have a color device. Color support has not been released yet as far as I know but will be coming next release also hopefully for you.

If you want to test out the color support you can apply the patch in the following comment and hopefully it will work for you:

https://github.com/koreader/koreader/pull/8727#issuecomment-1023019728

**Comment by berdik88** [comment]
Hi @rjd22 !
Thank you, kind man, for your time and effort!!! ))) 
The patch worked on PocketBook 633 Color. 

[It was](https://disk.yandex.ru/i/HbcWR42DxWj5qQ). 
[Become](https://disk.yandex.ru/i/UH8gEbE6jhwFWg). 

PS: now wait for a full implementation in the next release (the night mode and some other functions are cut out in the test)?

**Reply by berdik88** [reply]
I am happy to support testing on my 633, if necessary.

**Reply by rjd22** [reply]
Night mode and rotation will also work in the next release. They were fixed after this patch.

**Comment by junioryo** [comment]
No clue gang
